bunch-git-hook/src/main/kotlin/org/jetbrains/bunches/precommitHook/BunchPreCommitHook.kt

```diff
@@ -0,0 +1,54 @@
+package org.jetbrains.bunches.precommitHook
+
+import org.eclipse.jgit.api.Git;
+import java.io.File;
+import java.lang.System.exit
+import java.util.*
+
+fun getBunchExtensions(dotBunchFile: File): Set<String>? {
+    val lines = dotBunchFile.readLines().map { it.trim() }.filter { !it.isEmpty() }
+    if (lines.size <= 1) return null
+
+    return lines.drop(1).map { it.split('_').first() }.toSet()
+}
+
+fun main(args: Array<String>) {
+    if (args.size !== 0) {
```

Comment:
Please mind inspections IntelliJ reports. Identity equals shouldn't be used here.